core/common/library/Media/MediaFiles.php

```diff
@@ -12,8 +12,8 @@
  */
 namespace Phanbook\Media;
 
-use League\Flysystem\Filesystem;
-use League\Flysystem\Adapter\Local as Adapter;
+use \League\Flysystem\Filesystem;
+use \League\Flysystem\Adapter\Local as Adapter;
```

Comment:
The leading backslash is unnecessary and not recommended, as import names must be fully qualified, and are not processed relative to the current namespace.